config/ember-try.js

```diff
@@ -53,7 +53,8 @@ module.exports = function() {
             devDependencies: {
               'ember-source': urls[1]
             }
-          }
+          },
+          allowedToFail: true
```

Comment:
I understand allowing canary to fail, but do we want beta releases to get a pass as well?  Just trying to understand the thinking here.

Reply:
Beta has inherited the same error that caused canary to fail.